### PR TITLE
Deploy to dokku using patched action

### PR DESCRIPTION
**Pull request by slavakirichenko**
* Use a patched action which supports deploying to multiple environments.
* Updated checkout to v3.

### CODE REVIEW

File: .github/workflows/deploy.yml [hunk 1]
```diff
@@ -9,12 +9,12 @@ jobs:
   deploy:
     runs-on: ubuntu-20.04-self-hosted
     steps:
-      - uses: actions/checkout@v2
+      - uses: actions/checkout@v3
```
Comment (LeonidVas) [inline .github/workflows/deploy.yml:12]: In fact this change is not related with commit title.
I think one of two would be nice:
- replace the commit title with a more common
- split a commit

**Review by LeonidVas — CHANGES_REQUESTED** [review]
Hi! Thank you for the patch.
We are use commit decoration described in  https://github.com/tarantool/tarantool/wiki/Code-review-procedure#commit-message
"cd: patch the dokku deploy action"

**Comment by LeonidVas** [comment]
Update the CHANGELOG

File: .github/workflows/deploy.yml (hunk 1 above)
Comment (slavakirichenko) [reply .github/workflows/deploy.yml:12]: Changed the commit title and added information in CHANGELOG

Comment (ylobankov) [reply .github/workflows/deploy.yml:12]: Let's have this change in a separate commit with a description of why you do it.

**Review by ylobankov — CHANGES_REQUESTED** [review]

**Comment by ylobankov** [comment]
> Update the CHANGELOG

Should we really add any entries about CI changes to changelog? I think it is not interesting to users. Such changes don't affect users at all, it is our internal stuff.

File: .github/workflows/deploy.yml (hunk 1 above)
Comment (slavakirichenko) [reply .github/workflows/deploy.yml:12]: Done

File: CHANGELOG.md [hunk 2]
```diff
@@ -14,6 +14,9 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
   fixes for handling rpm repository metainformation. The most
   significant of these is the addition of the ability to remove
   information about a removed package.
+- Changed version of dokku deploy action to patched one which supports
```
Comment (LeonidVas) [inline CHANGELOG.md:17]: Nice try, but no.
See: https://github.com/tarantool/rws/commit/7a12d43ff47e90ef991e37857ed6a3a9b2ef75f1#diff-06572a96a58dc510037d5efa622f9bec8519bc1beab13c9f251e97e657a9d4ed

**Review by LeonidVas — CHANGES_REQUESTED** [review]

**Comment by LeonidVas** [comment]
> > Update the CHANGELOG
> 
> Should we really add any entries about CI changes to changelog? I think it is not interesting to users. Such changes don't affect users at all, it is our internal stuff.

Usually no.
But I have several points:
- RWS users are us, and it's not so much about RWS as it is about making it work properly in our infrastructure
- this is the only change that will cause us to do a release

File: CHANGELOG.md (hunk 2 above)
Comment (slavakirichenko) [reply CHANGELOG.md]: Fixed

**Review by LeonidVas — APPROVED** [review]

**Review by ylobankov — APPROVED** [review]
LGTM
